packages/skill-lesson/path-to-purchase/pricing.tsx

```diff
@@ -243,6 +246,7 @@ export const Pricing: React.FC<React.PropsWithChildren<PricingProps>> = ({
   // do not show the box if it's a downgrade
   const showPPPBox =
     isPPPEnabled &&
+    !purchaseToUpgrade &&
```

Comment:
we really need an additional "ppp upgrade box" or something becuase this was dangling and weird when upgrade was available so this makes it go away

possible side effect: upgrading with PPP from lower tier to higher tier breaks? 😭